### ISSUE TITLE
Request: insert pdf into note

### ISSUE THREAD

**Issue by kimaldis** [HappenApps/Quiver#882]
At the moment, dragging a pdf file onto Quiver inserts a link. The request is to have it add the pdf as a viewable note or cell.

thanks.

**Comment by kimaldis** [comment]
Maybe PDF cells?

**Comment by ylian** [comment]
No, it should not be a cell. Quiver should support inline PDF in text cells (both editor and preview), and Markdown cells (preview only).

**Comment by Kelym** [comment]
Agree, that would be a great feature to have! Currently I am switching to other document management library (Evernote, Skim, Highlight) for annotating PDF and commenting. Would be thrilled to see if Quiver could render PDF in cells and allow user to add a markup layer of comment on top of it.

**Comment by ppatel26** [comment]
@ylian Any idea when this would be introduced? :)

**Comment by kimaldis** [comment]
quietly bumping this.

**Comment by PatrickF1** [comment]
Bump. With this feature I will be 100% ready to switch from Evernote to Quiver.

**Comment by AdamBD** [comment]
Would love to see this!

**Comment by jiangchuanheglobal** [comment]
same here, is this feature available now ? I feel it can do something like the 1password which is able to save files to its own folder.

**Comment by AdamBD** [comment]
Is there a way to escalate features?

**Comment by jxshen311** [comment]
This would be a very nice feature!

**Comment by isehgu** [comment]
Bump. Love to have that feature.
